### PR TITLE
ROC-3285 Integrate nodejs logging with azure app

### PR DESCRIPTION
**Pull request by dharmendrak**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "@hmcts/draft-store-client": "^1.1.1",
     "@hmcts/info-provider": "^1.0.0",
     "@hmcts/nodejs-healthcheck": "^1.4.5",
-    "@hmcts/nodejs-logging": "^2.2.0",
+    "@hmcts/nodejs-logging": "file:./../../hmcts/nodejs-logging",
```
Comment (timja-kainos) [inline package.json]: `yarn link @hmcts/nodejs-logging` is a better way of doing this, it doesn't change the package.json file instead it removes the directory in your node_modules and adds a symlink
`yarn unlink @hmcts/nodejs-logging && yarn add @hmcts/nodejs-logging` when you are done working locally on it

Comment (dharmendrak) [reply package.json]: done

**Comment by rlewan** [comment]
There is a change to `yarn.lock` but no corresponding change to `package.json`.

**Comment by geek-soft** [comment]
@dharmendrak please pint `package.json` to `^3.0.0` version of the logging library.

**Comment by geek-soft** [comment]
@rlewan and @timja-kainos, I would love your reviews?

@timja-kainos you approved legal changes, here we have the same ones.

**Pull request merged**